libs/sdk-backend-spi/src/workspace/execution/results.ts

```diff
@@ -163,6 +164,29 @@ export interface IResultTotalHeader {
  */
 export type IResultHeader = IResultAttributeHeader | IResultMeasureHeader | IResultTotalHeader;
 
+/**
+ * Represents an execution result warning.
+ * (e.g. when execution was executed successfully, but backend didn't take something into the account)
+ *
+ * @public
+ */
+export interface IResultWarning {
+    /**
+     * Unique identifier of the execution warning
+     */
+    warningCode: string;
+
+    /**
+     * Human readable representation of the execution warning
+     */
+    message: string;
+
+    /**
+     * Execution warning parameters (e.g. when filter was not applied - its ObjRef)
+     */
+    parameters?: (ObjRef | boolean | number | string | null)[];
```

Comment:
How are the parameters related to the message in SPI (in Bear there are some placeholders in the message right?). Is this the API we want for SPI? If so, we should probably comment on this here (like what format do the placeholdres have in the message, etc.)